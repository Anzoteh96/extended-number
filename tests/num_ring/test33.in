-684449225029158310994215049026246334141581202697268347555834739735151703656263053696923991244078211
-681058267968692711120151886917931974515733241605174526299557359010543537899266987350809121802530153
-297477308200157374511308316903629897149139671628872342745203081068828158888178324829638245851324712
