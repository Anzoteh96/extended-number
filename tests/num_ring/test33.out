Enter the number base b:
Enter two numbers n and m:
b is 684449225029158310994215049026246334141581202697268347555834739735151703656263053696923991244078211
n mod b is 3390957060465599874063162108314359625847961092093821256277380724608165756996066346114869441548058
m mod b is 386971916829000936482906732122616436992441531068396004810631658666323544768084728867285745392753499
n + m is 390362873889466536356969894230930796618289492160489826066909039390931710525080795213400614834301557
n - m is 300868265260622974385371479011944256774987632720966164001480461793436324645174391175753115292872770
n * m is 523828549222396705383241818592136676405343092029571867465805968615476475026529289780246956937342683
n ** m is 274181658078557582421074172726515004072481125446529734812281336299839129077896362494035332718790799
